### DISCUSSION TITLE
But how on earth do you install extensions on remote server with 0.164.2

### DISCUSSION THREAD

**Discussion by SyntheticBird45** [zed-industries/zed#21575] [Feature Requests]
I can't find it. and referenced commit don't explain how.

**Comment by 5brian** [comment]
See: https://github.com/zed-industries/zed/issues/20074#issuecomment-2452051502

**Reply by SyntheticBird45** [reply]
That doesn't answer my question...

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
